Initium-Odp/src/com/universeprojects/miniup/server/controllers/ViewItemController.java

```diff
@@ -308,6 +309,44 @@ else if (iconUrl!=null && iconUrl.startsWith("http")==false)
 			itemMap.put("strmod", field.toString());
 		}
 		
+		field = item.getProperty("Slotted:maxCount");
+		if (field!=null && field.toString().trim().equals("")==false)
+		{
+			requirements=true;
+			itemMap.put("slotCount", field.toString());
+			
+			List<EmbeddedEntity> slotItems = (List<EmbeddedEntity>) item.getProperty("Slotted:slotItems");
+			List<String> itemSlots = new ArrayList<String>();

```

Comment:
This is unnecessary and I'm not sure you'll need it, you already seem to have the datastructures you need.